app/models/manageiq/providers/autosde/storage_manager/host_initiator.rb

```diff
@@ -5,15 +5,17 @@ class ManageIQ::Providers::Autosde::StorageManager::HostInitiator < ::HostInitia
   def self.raw_create_host_initiator(ext_management_system, options = {})
     wwpns = Array(options['custom_wwpn'])
     wwpns += Array(options['wwpn']).map { |item| item["value"] }
-
-    # wwpn values send to autosde should be format as colon separated string (e.g. WWPN1:WWPN2:WWPN3)
```

Comment:
Maybe better to move this comment up above `wwpns` here https://github.com/ManageIQ/manageiq-providers-autosde/pull/160/files#diff-41fa59bd46caf3a422d2250625c4a970d5a0eb9840ba37722b2a73b1407b171aR6 since it applies to wwpns and iqns

Also, should be `wwpn values sent` not `send` to autosde

Reply:
I modified the comments as you suggested.